Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM java:openjdk-7-jdk
+
+RUN apt-get update
+ENV CLOUD_SDK_VERSION 161.0.0
+RUN apt-get -qqy update && apt-get install -qqy \
+        curl \
+        gcc \
+        python-dev \
+        python-setuptools \
+        apt-transport-https \
+        lsb-release \
+        openssh-client \
+        git \
+    && easy_install -U pip && \
+    pip install -U crcmod   && \
+    export CLOUD_SDK_REPO="cloud-sdk-$(lsb_release -c -s)" && \
+    echo "deb https://packages.cloud.google.com/apt $CLOUD_SDK_REPO main" > /etc/apt/sources.list.d/google-cloud-sdk.list && \
+    curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add - && \
+    apt-get update && \
+    apt-get install -y google-cloud-sdk=${CLOUD_SDK_VERSION}-0 \
+        google-cloud-sdk-app-engine-python \
```

Comment:
See comment above (we could pare this down but keeping it this way makes this the equivalent of what Google has defined in their own Dockerfile, so you get the same behavior as `FROM google/cloud-sdk`)